src/families/hedera/js-receive.ts

```diff
@@ -0,0 +1,65 @@
+import { WrongDeviceForAccount } from "@ledgerhq/errors";
+import { Observable } from "rxjs";
+import { close, open } from "../../hw";
+import { Account, isSegwitDerivationMode } from "../../types";
+import getAddress from "../../hw/getAddress";
+
+export default function receive(
```

Comment:
Same concern as for the scan & sync account, the default receive should be used preferably... but it's on us to help you find a way to fit our model.